egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -104,10 +104,16 @@ def GetArgs():
     parser.add_argument("--relu-dim", type=int,
```

Comment:
@pegahgh You code is undoing most of the recent changes. Please update your branch.